src/js/evented.js

```diff
@@ -26,38 +26,29 @@ export class Evented {
     if (isUndefined(handler)) {
```

Comment:
perhaps we should have a shared NPM package for Evented? Seems basically the same as https://github.com/shipshapecode/shepherd/blob/master/src/js/evented.js